### PR TITLE
Add an argument ignoreBuiltinModules to the 'get visible symbols' prim

### PR DESCRIPTION
**Pull request by daviditen**
When true, it skips printing symbols in the Builtin, Types, and Math modules.

Updated some errors to allow for multiple arguments to this primitive.

Adjusted some tests to use the new argument and stop using --minimal-modules.

### CODE REVIEW

**Review by lydia-duncan — APPROVED** [review]
Minor things that you can take or leave

File: compiler/passes/scopeResolve.cpp [hunk 1]
```diff
@@ -2772,31 +2772,74 @@ static void lookupAndAddToVisibleMap(const char* name, CallExpr* call,
...
+static void errorForUnexpectedArgName(std::vector<std::string> argNames,
+                                      CallExpr* call) {
+  if (((unsigned)call->numActuals()) > argNames.size()) {
+    USR_FATAL(call, "too many arguments to 'get visible symbols'");
+  }
+
+  for (int i = 1; i <= call->numActuals(); i++) {
+    NamedExpr* actual = toNamedExpr(call->get(i));
+    if (!actual) {
+      USR_FATAL(call, "'get visible symbols' requires named arguments");
```
[the remark this answers is not in the document]
Comment (daviditen) [reply compiler/passes/scopeResolve.cpp:2817]: They didn't have to be, but adding the ability to make them unnamed and positional would have added complexity and IMO reduced the clarity of calling this. Forcing named arguments felt like the right balance.

File: compiler/passes/scopeResolve.cpp [hunk 2]
```diff
@@ -2772,31 +2772,74 @@ static void lookupAndAddToVisibleMap(const char* name, CallExpr* call,
...
+  expectedNames.push_back((std::string)name);
+
+  for (int i = 1; i<= call->numActuals(); i++) { 
+    NamedExpr* ne = toNamedExpr(call->get(i));
+    if (ne && !strcmp(ne->name, name)) {
+      SymExpr* se = toSymExpr(ne->actual);
+      if (se && (se->symbol() == gTrue || se->symbol() == gFalse)) {
+        ret = se->symbol() == gTrue;
+      } else {
+        USR_FATAL(se, "the arguments to 'get visible symbols' must be literals 'true' or 'false'");
```
[the remark this answers is not in the document]
Comment (daviditen) [reply compiler/passes/scopeResolve.cpp]: Updated to "must be either ..." wording.

**Comment by daviditen** [comment]
Thanks!

**Pull request merged**